src/app/directives/form-connector/form-connector.directive.ts

```diff
@@ -21,14 +21,9 @@ export class FormConnectorDirective implements OnInit, OnDestroy {
   ngOnInit(): void {
     this.formChange = this.formGroupDirective.form.valueChanges
       .pipe(
-        debounceTime(this.debounce),
-        tap((value) => {
-          console.log('VALUE:', value, 'PATH:', this.path);
-        })
+        debounceTime(this.debounce)
       )
       .subscribe();
-
-    console.log(this.formGroupDirective.form);
```

Comment:
@JonLazariniDD  this directive was written mainly as diagnosis tool. It helps in understanding how the parent formGroup behaves down to the nth component. Once all the big tickets are done and settled the directive job will be done. Please lets leave for now and remove later